doc/mgmt/SONiC_IntfRangeFeature_HLD.md

```diff
@@ -0,0 +1,235 @@
+end             Exit to the exec Mode
+exit            Exit from current mode
+link            Interface link                   
+mac             Configure MAC parameters
+mtu             Configure MTU        
+nat-zone        NAT Zone               
+no              Negate a command or set its defaults
+service-policy  Apply ingress or egress policy      
+sflow           Interface sFlow settings
+shutdown        Disable the interface     
+spanning-tree   Configure spanning tree on interface      
+storm-control   configure storm-control   
+switchport      Configure switchport parameters      
+threshold       Configure priority-group/queue threshold on an interface.
+udld            UDLD interface configuration subcommands
+
+sonic(conf-if-range-Eth1-4,5)# mtu 9000
+sonic(conf-if-range-Eth1-4,5)#
+```
+####  3.6.2.1.2 Vlan interface range:
+###### Configure Vlan interfaces existing in given range
+Syntax: `interface range Vlan Vlan-id - Vlan-id[,Vlan-id - Vlan-id...]`
+```
+sonic(config)# interface range Vlan 1-20
+sonic(conf-if-range-Vl-1-20)# ?
+end                Exit to the exec Mode
+exit               Exit from current mode
+link               Interface link             
+mac                Configure MAC parameters   
+mtu                Configure MTU
+nat-zone           NAT Zone   
+neigh_suppress     Enable ARP and ND Suppression      
+no                 Negate a command or set its defaults
+service-policy     Apply ingress or egress policy
+
+sonic(conf-if-range-Vl-1-20)# mtu 9000
+sonic(conf-if-range-Vl-1-20)#
+```
+###### Create Vlan interfaces in given range
+Syntax: `interface range create Vlan Vlan-id - Vlan-id[,Vlan-id - Vlan-id...]`
+```
+sonic(config)# interface range create ?
+Vlan             Vlan interface range
+PortChannel      PortChannel interface range
+
+sonic(config)# interface range create Vlan 3-4,10,11
+sonic(conf-if-range-Vl-3-4,10,11)#
+```
+###### Delete Vlan interfaces existing in given range
+Syntax: `no interface Vlan Vlan-id - Vlan-id[,Vlan-id - Vlan-id...]`
+```
+sonic(config)# no interface Vlan 1-20
+sonic(config)#
+```
+#### 3.6.2.1.3 PortChannel interface range:
+###### Configure PortChannel interfaces existing in given range
+Syntax: `interface range PortChannel Po-id - Po-id[,Po-id - Po-id...]`
+```
+sonic(config)# interface range PortChannel 1-40
+sonic(conf-if-range-Po-1-40)# ?
+end             Exit to the exec Mode
+exit            Exit from current mode
+link            Interface link          
+mac             Configure MAC parameters   
+mclag           Configure MCLAG interface  
+mtu             Configure MTU    
+nat-zone        NAT Zone         
+no              Negate a command or set its defaults
+service-policy  Apply ingress or egress policy     
+shutdown        Disable the interface              
+spanning-tree   Configure spanning tree on port-channel   
+switchport      Configure switchport parameters          
+
+sonic(conf-if-range-Po-1-20)# mtu 9000
```

Comment:
Can you please mention the error in the following case?

sonic(config)# interface range PortChannel 1-40
sonic(conf-if-range-Po-1-40)# Ip address 10.1.1.1/24